WMCDuplicateRemover.Tests/EPG/EPGIntegrationTests.cs

```diff
@@ -17,7 +13,9 @@ public class EPGIntegrationTests
         [TestFixtureSetUp]
         public void SetUp()
         {
-            tv = new XmlTvEpgWrapper(@"..\..\EPG\TestData\EPG.xml");
+            var dateTimeMock = new Mock<IDateTime>();
```

Comment:
Can you explain in simple language what an interface is?  Also, do you only mock things for tests? or is there a reason you'd do it outside of a test?  You wouldn't right?  Because when the code is just running running it'll either work or it won't, right?